cpp-ch/local-engine/Parser/AggregateRelParser.cpp

```diff
@@ -289,14 +289,16 @@ void AggregateRelParser::addAggregatingStep()
     if (enable_streaming_aggregating)
     {
         // Disable spilling to disk.
+        // We let group_by_two_level_threshold_bytes = 0, avoid converting the hash table into two level
+        // in Aggregator when the hash table is not large enough.
         Aggregator::Params params(
             grouping_keys,
             aggregate_descriptions,
             false,
             settings.max_rows_to_group_by,
             settings.group_by_overflow_mode,
             settings.group_by_two_level_threshold,
```

Comment:
what about hash table size?

Reply:
The size limit of hash table does not cause this problem. `Aggregator` will convert a hash table into a two level one when the memory usage the hash table  reaches the limit. But the memory usage statistics is not right